cmd/scollector/collectors/chef_linux.go

```diff
@@ -0,0 +1,139 @@
+package collectors
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"bufio"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+type CRSummary struct {
+	StartTime   string  `json:"start_time"`
+	EndTime     string  `json:"end_time"`
+	ElapsedTime float32 `json:"elapsed_time"`
+	Status      string  `json:"status"`
+}
+
+func init() {
+	collectors = append(collectors, &IntervalCollector{F: c_chef_linux, Enable: chefEnable})
+}
+
+const (
+	chefRunLog = "/var/chef/reports/run_status_json.log"
+)
+
+func chefEnable() bool {
+	_, err := os.Stat(chefRunLog)
+	return err == nil
+}
+
+func c_chef_linux() (opentsdb.MultiDataPoint, error) {
+	md, err := chef_linux(chefRunLog)
```

Comment:
Additional function is for tests convenience. Will include tests later, need to discuss a bit.